Slambook/index.js

```diff
@@ -54,30 +55,8 @@ form.addEventListener("submit", (e) => {
 
     // Display the data
     console.log(allSlamArray);
-
-    const allSlams = document.querySelector('.allSlams');
-   allSlamArray.forEach((item) => {
-     // Create a new element for each item
-     const element = document.createElement("div");
-
-     // Set innerHTML or textContent of the element based on the data
-     element.innerHTML = `
-      <p>Contact Name: ${item.contactName}</p>
-      <p>Relationship: ${item.relationship}</p>
-      <p>Something You Like in Me: ${item.likeInMe}</p>
-      <p>Something You Hate in Me: ${item.hateInMe}</p>

```

Comment:
Consider adding some spacing or line breaks between the paragraphs for better readability.